packit_service/worker/jobs.py

```diff
@@ -459,17 +445,39 @@ def process_message(
                 ),
             )
         ):
-            jobs_results = self.process_comment_jobs(event_object)
+            processing_results = self.process_comment_jobs(event_object)
         else:
             # Processing the jobs from the config.
-            jobs_results = self.process_jobs(event_object)
+            processing_results = self.process_jobs(event_object)
 
-        logger.debug("All jobs finished!")
+        return processing_results or get_processing_results(event=event_object, jobs=[])
 
-        task_results = {"jobs": jobs_results, "event": event_object.get_dict()}
 
-        for v in jobs_results.values():
-            if not (v and v["success"]):
-                logger.warning(task_results)
-                logger.error(v["details"]["msg"])
-        return task_results
+def get_signature(task_name: str, event: Event, job: Optional[JobConfig]):
```

Comment:
What about having this as a `JobHandler` class method?

Then you'd use it as
```python
  signatures = [handler_kls.get_signature(event=event, job=job) for job in jobs]
  group(signatures).apply_async()
```
instead of
```python
  signatures = [
      get_signature(task_name=handler_kls.task_name, event=event, job=job)
      for job in jobs
  ]
  group(signatures).apply_async()
```

Reply:
thanks, changed